lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -27,11 +29,20 @@
 
 MAX_ID_RANGE = 252
 
+# Maximal joints range after applying calibration
```

Comment:
Could you either add more to this comment or point us to where there is documentation. I'm not sure what it means.

Reply:
Thanks @Cadene. Now I understand :) Do you think we should use more common terms like:
- "joints in degree" -> "revolute joints"
- "joints in percentage" -> "linear joints" or "prismatic joints"